packages/admin/dashboard/src/lib/promotions.ts

```diff
@@ -1,31 +1,49 @@
 import { HttpTypes } from "@medusajs/types"
+import { i18n } from "../components/utilities/i18n"
 
 export enum PromotionStatus {
   SCHEDULED = "SCHEDULED",
   EXPIRED = "EXPIRED",
   ACTIVE = "ACTIVE",
-  DISABLED = "DISABLED",
+  INACTIVE = "INACTIVE",
+  DRAFT = "DRAFT",
+}
+
+export type StatusColors = "grey" | "orange" | "green" | "red" | "grey"
+export type StatusMap = Record<string, [StatusColors, string]>
+export const promotionStatusMap: StatusMap = {
+  [PromotionStatus.ACTIVE]: ["orange", i18n.t("statuses.active")],
```

Comment:
todo: Should be green

Reply:
@olivermrbl addressed the cache and color issues 🤞🏻